packages/components/src/focal-point-picker/test/index.js

```diff
@@ -76,12 +75,10 @@ describe( 'FocalPointPicker', () => {
 					} }
 				/>
 			);
-			// Click and press arrow up
 			const dragArea = getByRole( 'button' );
-			act( () => {
-				fireEvent.mouseDown( dragArea );
-				fireEvent.keyDown( dragArea, { charCode: 0, keyCode: 38 } );
-			} );
+			fireEvent.mouseDown( dragArea, { clientX: 1, clientY: 1 } );
+			// Change doesn't fire until mouseup
+			fireEvent.mouseUp( dragArea );
```

Comment:
Just that I recalled the test had originally been attempted with mouse events (https://github.com/WordPress/gutenberg/pull/38247#discussion_r803154354) and thought it'd good to try since it's probably the more expected way to test it. I think I'd also imagined it could help as a starting place in case a more a in-depth mouse based test was devised 🤷.